Contrast-Security-OSS/contrast-intellij-plugin
# TS-21864 upgrade gradle wrapper

# Overview
The next intellij plugin version needs gradle 7.3 or greater. This is to update the gradle wrapper to use gradle 7.6

# What changed?
1. Updated gradle wrapper files to 7.6
2. Updated readme to reflect previous changes.

# How to test?
1. Delete the build folder to get rid of the old artifacts
2. run `./gradlew runIde`
3. This will open an intellij instance with the plugin installed
4. open a project
5. In the menu bar go to View>Tools Windows>Contrast
6. This will open the Contrast window
7. Click on the gear icon and fill out the connection information > Add and apply
8. Click on the filter icon and select an application
9. Verify that you are able to see the assess results.
